BankSDK/GiniBankSDKExample/Tests/SettingsViewControllerTests.swift

```diff
@@ -711,17 +731,43 @@ extension SettingsViewControllerTests {
 	}
 	
 	func testSegmentedControlPDFAndImages() {
-		guard let index = getFileImportOptionIndex()else {
+		guard let index = getFileImportOptionIndex() else {
 			XCTFail("`fileImportType` option not found in sectionData")
 			return
 		}
-		guard case .fileImportType(var data) = contentData[index] else { return }
+		guard case .segmentedOption(var data) = contentData[index] else { return }
 		data.selectedIndex = 2
 		configuration.fileImportSupportedTypes = giniImportFileType(selectedIndex: data.selectedIndex)
 		XCTAssertEqual(configuration.fileImportSupportedTypes,
 					   .pdf_and_images,
 					   "pdf and image types should be supported in the gini configuration")
 	}
+    
+    func testButtonEntryPoint() {
+        guard let index = getEntryPointIndex() else {
+            XCTFail("`entryPoint` option not found in sectionData")
+            return
+        }
+        guard case .segmentedOption(var data) = contentData[index] else { return }
```

Comment:
[suggestion] We should also add `, data.optionType == .entryPoint` as an extra condition in guard. In both methods and also for test methods for `data.optionType == .fileimport`